documentation/library_guide/dynamic_selection_api/fixed_resource_policy.rst

```diff
@@ -158,87 +147,55 @@ and then remain constant.
 Constructors
 ------------
 
-``fixed_resource_policy`` provides three constructors.
+``fixed_resource_policy`` provides four constructors.
 
 .. list-table:: ``fixed_resource_policy`` constructors
   :widths: 50 50
   :header-rows: 1
-  
+
   * - Signature
     - Description
   * - ``fixed_resource_policy(deferred_initialization_t);``
     - Defers initialization. An ``initialize`` function must be called prior to use.
-  * - ``fixed_resource_policy(std::size_t offset = 0);``
+  * - ``fixed_resource_policy(std::size_t index = 0);``
     - Sets the index for the resource to be selected. Uses the default set of resources.
-  * - ``fixed_resource_policy(const std::vector<resource_type>& u, std::size_t offset = 0);``
+  * - ``fixed_resource_policy(const std::vector<resource_type>& u, std::size_t index = 0);``
     - Overrides the default set of resources and optionally sets the index for the resource to be selected.
+  * - ``fixed_resource_policy(const std::vector<resource_type>& u, ResourceAdapter adapter, std::size_t index = 0);``
+    - Overrides the default set of resources with a resource adapter and optionally sets the index for the resource to be selected.
 
 Deferred Initialization
 -----------------------
 
-A ``fixed_resource_policy`` that was constructed with deferred initialization must be 
-initialized by calling one its ``initialize`` member functions before it can be used
+A ``fixed_resource_policy`` that was constructed with deferred initialization must be
+initialized by calling one of its ``initialize`` member functions before it can be used
 to select or submit.
 
-.. list-table:: ``fixed_resource_policy`` constructors
+.. list-table:: ``fixed_resource_policy`` initializers

```

Comment:
I would remove all the table titles. These seem completely redundant at this page, as each table comes in a separate subsection with a heading.